### DISCUSSION TITLE
Install Twill on existing laravel application

### DISCUSSION THREAD

**Discussion by mariyaartsy** [area17/twill#2172] [General]
Hi Team,

I am new to Twill CMS in laravel. I have one working laravel application with jetstream admin panel.
Anyone please tell me how can I install Twill for CMS static pages and blog pages in my application?

Best Regards,
Mariya

**Comment by kallefrombosnia** [comment]
Hello, you shouldn't have problems integrating Twill CMS into your existing Laravel application.

You can check installation steps and other development docs here: [Docs](https://twill.io/docs/3.x/getting-started/installation.html)
Note: this is 3.x (docs) which is not yet a stable release, but you freely use it to start developing, soon stable version will be released.

The latest version of 3.x
**composer require area17/twill:"3.0.0-rc4"**

Let us know if you have any other questions or chat with us directly on the [Discord server](https://discord.gg/cnWk7EFv8R).
